lib/gitlab/client/wikis.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+class Gitlab::Client
+  # Defines methods related to wikis.
+  # @see https://docs.gitlab.com/ce/api/wikis.html
+  module Wikis
+    # Get all wiki pages for a given project.
+    #
+    # @example
+    #   Gitlab.wikis(3)
+    #   Gitlab.wikis(3, {with_content: 'Some wiki content'})
+    #
+    # @param  [Integer, String] project The ID or name of a project.
+    # @param  [Hash] options A customizable set of options.
+    # @option options [String] :with_content(optional) Include pages content
```

Comment:
Is `:with_content` supposed to be a symbol?

Reply:
It looks like it is, but the other options in this file aren't symbols, that should probably be fixed :)